web/public/login.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
```

Comment:
Gonna be honest, not sure how this happened. git does some weird stuff with merging sometimes. I was just trying to catch this branch up to web-dev to get rid of merge conflicts